src/services/db/index.ts

```diff
@@ -54,25 +54,7 @@ export default class DBService extends GenericService {
       logging: false,
     });
 
-    if (!config.dbDialect || config.dbDialect === 'sqlite') {
-      this.sequelize = new Sequelize({
-        dialect: 'sqlite',
-        storage: config.dbStorage,
-        logging: false,
-      });
-    } else {
-      this.sequelize = new Sequelize(
-        config.dbName as string,
-        config.dbUsername as string,
-        config.dbPassword,
-        {
-          host: config.dbHost,
-          port: Number(config.dbPort),
-          dialect: config.dbDialect as Dialect,
-          logging: false,
-        }
-      );
-    }
+    this.sequelize = sequelize;
```

Comment:
This was creating new Sequelize instances every time. And Sequelize always leaves the db connections open unless explicitly [closed](https://sequelize.org/docs/v6/getting-started/#closing-the-connection).

Instead, `sequelize` from `util` returns the cached instance and avoid creating new connections.